jvc_projector/__init__.py

```diff
@@ -55,17 +59,22 @@ class ACKs(Enum):
 class JVCProjector:
     """JVC Projector Control"""
 
-    def __init__(self, host, port = 20554):
+    def __init__(self, host, port=20554):
         self.host = host
         self.port = port
+        self.delay = 0.6
 
     def _send_command(self, operation, ack=None):
         JVC_GREETING = b'PJ_OK'
         JVC_REQ = b'PJREQ'
         JVC_ACK = b'PJACK'
+        result = False
 
         jvc_sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-        jvc_sock.connect((self.host, self.port)) # connect to projector
+        try:
+            jvc_sock.connect((self.host, self.port)) # connect to projector
+        except:
+            return "unknown"
```

Comment:
If `jvc_sock.connect()` fails for some reason, it should return an informative error message and cancel code execution. I let the socket exceptions speak for themselves, instead of implementing my own exceptions over the top.

Your implementation catches the socket error and returns a perfectly viable message "unknown" instead of raising an exception. Functions which invoke `_send_command` will not know that a socket exception occurred and code execution will try to continue (unless you write code to check for an `unknown` return message). I believe the original implementation is more intuitive.